07-errors.Rmd

```diff
@@ -0,0 +1,196 @@
+# Error messages
+
+```{r}
+knitr::opts_chunk$set(eval = FALSE)
+```
```

Comment:
Does this chunk need `include = FALSE` to hide the code?

Reply:
Oops